src/njoy21/input/POWR/test/POWR.test.cpp

```diff
@@ -0,0 +1,516 @@
+#define CATCH_CONFIG_MAIN
+
+#include "catch.hpp"
+
+#include "njoy21.hpp"
+
+using namespace njoy::njoy21::input;
+
+SCENARIO( "POWR",
+          "[POWR]" ){
+  GIVEN( "valid inputs" ){
+    WHEN( "using lib=1" ){
+      std::string inp(
+ std::string( " 27 29 /\n" ) + // Card1
```

Comment:
This probably constitutes the first time that POWR has been run in 30+ years!

Reply:
Technically, the FORTRAN remains untouched, so I'm not as confident about that.  Heh.